Lib/test/test_binascii.py

```diff
@@ -117,6 +114,51 @@ def addnoise(line):
         # empty strings. TBD: shouldn't it raise an exception instead ?
         self.assertEqual(binascii.a2b_base64(self.type2test(fillers)), b'')
 
+    # TODO: RUSTPYTHON
+    @unittest.expectedFailure
```

Comment:
Please break these lines out into a separate commit.

Reply:
I separated these lines in https://github.com/RustPython/RustPython/pull/4344/commits/04537b1993035f78aaab94b12edc31e03d10882e commit. 🙏🏻